### PR TITLE
As a manager I should be able to resolve an incident, #38

### PR DESCRIPTION
**Pull request by ershovd**
- implement button in xaml
- implement ResolveCommand and ResolveAsync in view model
- add property for conditional show\hiding of resolve button
- add conditional enabling\disabling of buttons

Questions and blocking:
-  **blocking until API 28 included in dev**
-  **blocking until Employee role implemented (for conditional display of visibility)** or should we skip for now?
- **blocking until #37 is merged to dev - most likely I'll need to merge manually** 
- Question - should the manager be also able to fix the issue? I guess, yes, so he need to be able to see 3 buttons.

Currently, I've spend 1h on task (take some time testing), update estimate accordingly.

### CODE REVIEW

**Comment by j0nscalet** [comment]
>blocking until API 28 included in dev
This is done, kudos to @mgpaja8 

>blocking until Employee role implemented (for conditional display of visibility) or should we skip for now?
I hear what your saying this was an oversight on my part, let's skip this for now as the `IsManager` property does just fine.

>blocking until #37 is merged to dev - most likely I'll need to merge manually
Question - should the manager be also able to fix the issue? I guess, yes, so he need to be able to see 3 buttons.
This has been merged as well

File: QualityKitchen.Mobile/Application/ViewModels/IncidentResolutionPageViewModel.cs [hunk 1]
```diff
@@ -1,28 +1,49 @@
...
+
+
 		public IncidentResolutionPageViewModel ()
 		{
 			Comment = String.Empty;
 			CommentButtonEnabled = false;
 			CommentContainerTranslationY = 0;
+
+			// TODO: this should be conditional, based on current employee role
+			IsVisibleForManager = true;
```
Comment (j0nscalet) [inline QualityKitchen.Mobile/Application/ViewModels/IncidentResolutionPageViewModel.cs]: You can bind this to Employee.IsManager

File: QualityKitchen.Mobile/Application/ViewModels/IncidentResolutionPageViewModel.cs [hunk 2]
```diff
@@ -51,28 +71,83 @@ public double CommentContainerTranslationY
                 return _commentContainerTranslationY;
             }
         }
-		
+
+		public bool IsVisibleForManager
```
Comment (j0nscalet) [inline QualityKitchen.Mobile/Application/ViewModels/IncidentResolutionPageViewModel.cs]: See above.

File: QualityKitchen.Mobile/Application/ViewModels/IncidentResolutionPageViewModel.cs (hunk 1 above)
Comment (ershovd) [inline QualityKitchen.Mobile/Application/ViewModels/IncidentResolutionPageViewModel.cs]: @j0nscalet where can I find `IsManager` property? the https://github.com/j0nscalet/QualityKitchen.Mobile/blob/dev/QualityKitchen.Mobile/Model/Employee.cs  class doesn't have it. but API class https://github.com/j0nscalet/QualityKitchen.API/blob/dev/src/QualityKitchen.Domain/Model/Operation/Employee.cs  do have.

Comment (j0nscalet) [inline QualityKitchen.Mobile/Application/ViewModels/IncidentResolutionPageViewModel.cs]: @ershovd That's my fault, go ahead and add the `IsManager`to `QualityKitchen.Mobile.Employee`.

**Comment by ershovd** [comment]
@j0nscalet `IIncidentServiceClient` methods `ResolveAsync` and `ResolveAsync` still doesn't have returned result with generic parameter as `Activity`. https://github.com/j0nscalet/QualityKitchen.Mobile/blob/dev/QualityKitchen.Mobile/Remote/Http/ServiceClients/Interfaces/IIncidentServiceClient.cs   
Please advise.

**Comment by j0nscalet** [comment]
@ershovd Those are on the client side, feel free to update those methods.

**Comment by ershovd** [comment]
-  merged the dev into feature#38 branch
- add InverterBooleanConverter - for conditional display of Fix and close butons
 - add IsManager property
spent 40m

Still needs to implement updated methods for IIncidentServiceClient

**Comment by ershovd** [comment]
@j0nscalet implement changes in  `IncidentServiceClient`interface and implementation, to match API signature.  also, add returned `Activity` from `FixAsync` and `ResolveAsync` methods in `IncidentResolutionPageViewModel`.

**Pull request merged**